route/include/route_generator_worker.h

```diff
@@ -291,6 +291,10 @@ namespace route {
         int cte_count_ = 0;
 
         int cte_count_max_;
+
+        // counter to record how many times an active route has been generated from a SetActiveRoute request without a Route ID
```

Comment:
When do we have routes without route names???

Reply:
PR #1321 provided new functionality that enabled port_drayage_plugin to process a received destination (latitude/longitude coordinate) from a MobilityOperation message and set a new active route to that destination point. The merged implementation did not assign a route name since no route .csv file was used, however, this could be updated to use some logical name since port_drayage_plugin does know the destination type that it is routing to.

Reply:
If updating port drayage would remove the need for this counter, I think that would be preferable. If the update would just move this counter to that node then this is fine as is.

Reply:
Made the fix in port_drayage_plugin that results in this counter being removed completely.